lib/Retcon/DataSource.hs

```diff
@@ -129,22 +136,17 @@ createDocument src doc = do
     liftIO $ hClose hin
 
     -- 4. Read handles
-    output <- T.filter (not. isSpace) . T.decodeUtf8 <$> (liftIO . BS.hGetContents $ hout)
-    err <- T.decodeUtf8 <$> (liftIO . BS.hGetContents $ herr)
+    (exit, output, err) <- liftIO $ gatherOutput (hproc, hout, herr)
+    let output' = T.filter (not . isSpace) $ T.decodeUtf8 output
```

Comment:
This code is only there because I didn't want to make a breaking change.

It should be trimming, not deleting all whitespace characters.